src/dotnet-install.ps1

```diff
@@ -901,6 +895,16 @@ Say "- The SDK needs to be installed without user interaction and without admin
 Say "- The SDK installation doesn't need to persist across multiple CI runs."
 Say "To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.`r`n"
 
+if ($Internal -and [string]::IsNullOrEmpty($FeedCredential)) {
+    Say-Warning "Note, it is necessary to use FeedCredential option when Internal switch is set."
```

Comment:
I think this should start with "Note that..."

Reply:
I would reword as "Applicable only when FeedCredential option is set"

Reply:
So, summarizing: "Note that Internal switch is applicable only when FeedCredential option is set."? To me it sounds a bit like the switch would be ignored otherwise, which is not the case, we will get a download error.  But this is still better, yes.

Reply:
I would just state that assumption:

```suggestion
    Say-Warning "Command was called with -Internal parameter, but no credentials were provided using -FeedCredential parameter. You will most likely get authorization errors."
```